app/views/layouts/nav/_staff_subnav.html.haml

```diff
@@ -11,7 +11,7 @@
             %i.fa.fa-info-circle
             %span Info
         %li
-          = link_to event_staff_event_teammate_invitations_path do
+          = link_to event_staff_teammate_invitations_path do
```

Comment:
we can remove _invitations from the path since this should be the teammate's index path